### ISSUE TITLE
UX of Community Page

### ISSUE THREAD

**Issue by techsmyth** [alkem-io/client-web#1181]
## Description

As a user, I want to be able to quickly understand and interact with the community around a Challenge.

## Acceptance criteria
- [x] show who's involved in the community
- - [x] members should be displayed
- - [x] organisations associated?
- [x] Members should be filterable
- [x] Be consistent with the navigation (members/organizations)
- [x] Add a link towards the 'Activity' page
- - [x] The link shouldn't be prominent (not main activity on the page)
- [x] Add capability to go back to the hub/challenge/opportunity
- [x] Display the name of the community

Should be left part of a different page:
- **Q&A Section, timeline, updates, etc. should **NOT** be included in the design**
- **communications - updates**
- **timeline / activity**

## Resulting design
https://xd.adobe.com/view/e0aa155d-63ef-4b01-8603-0ce36fc88072-b5f9/screen/6d0c6f4b-51b9-4e56-9316-4adc6e8e1d40/

### Notes
- Page 2 & 3 of the design show how organizations should be displayed on the hub / challenge communities.
- See inline comments.

**Comment by KayleeS96** [comment]
@techsmyth We have created the first version of the community page, please let us know your thoughts

![Community page@2x.png](https://images.zenhubusercontent.com/612cb1848a783db262f19451/9cc50bab-b963-4488-82c9-104f6bc08a3b)

**Comment by KayleeS96** [comment]
During our daily meeting, we decided to make multiple iterations to limit the white space on the page. We have made two new versions. One with 5 cards in a row with the same spacing as before and one with 5 cards in a row with minimal spacing between them.
![Community page – 5 Rows V1@2x.png](https://images.zenhubusercontent.com/612cb1848a783db262f19451/d5cacadc-55d1-4e9c-a516-a993a31aadb9)
![Community page – 5 rows V2, minimal spacing@2x.png](https://images.zenhubusercontent.com/612cb1848a783db262f19451/499d0b26-e088-41f9-acdf-7786f5576d34)
